datasync/downloader/downloader.go

```diff
@@ -799,14 +830,15 @@ func (d *Downloader) findAncestor(p *peerConnection, height uint64) (uint64, err
 // other peers are only accepted if they map cleanly to the skeleton. If no one
 // can fill in the skeleton - not even the origin peer - it's assumed invalid and
 // the origin is dropped.
-func (d *Downloader) fetchHeaders(p *peerConnection, from uint64, pivot uint64) error {
+func (d *Downloader) fetchHeaders(p *peerConnection, from uint64) error {
 	p.logger.Debug("Directing header downloads", "origin", from)
 	defer func(start time.Time) {
 		p.logger.Debug("Header download terminated", "elapsed", time.Since(start))
 	}(time.Now())
 
 	// Create a timeout timer, and the associated header fetcher
-	skeleton := true            // Skeleton assembly phase or finishing up
+	skeleton := true // Skeleton assembly phase or finishing up
```

Comment:
This is unnecessary change. Can we add more whitespace before the comment?

Reply:
This is also what `gofumpt` does. Maybe we can swap the line and the next one to align the comments.

Reply:
Ok. got it.